ember-phone-input/src/components/phone-input.ts

```diff
@@ -134,16 +204,16 @@ export default class PhoneInputComponent extends Component {
    * @type {Array}
    */
 
-  get onlyCountries() {
-    return this.args.onlyCountries || [];
+  get onlyCountries(): string[] {
+    return this.args.onlyCountries || Array<string>();
   }
 
   /**
    * Specify the countries to appear at the top of the list.
    * @argument preferredCountries
    * @type {Array}
    */
-  get preferredCountries() {
+  get preferredCountries(): string[] {
```

Comment:
Can this be `null` as well? Or an empty array must be passed instead?

Reply:
In case the argument is falsy it will return default value as an array.
`return this.args.preferredCountries || ['us', 'gb'];`

Reply:
In case the argument is falsy it will return default value as an array.
`return this.args.preferredCountries || ['us', 'gb'];`

Reply:
Makes sense, thank you!